pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2909,40 +2909,76 @@ var _ = Describe("Converter", func() {
 			Entry("VGA on ppc64le with EFI and BochsDisplayForEFIGuests set", ppc64le, v1.Bootloader{EFI: &v1.EFI{}}, true, "vga"),
 		)
 
-		DescribeTable("slic ACPI table should be set to", func(source v1.VolumeSource, isSupported bool, path string) {
-			slicName := "slic"
-			vmi.Spec.Domain.Firmware = &v1.Firmware{ACPI: &v1.ACPI{SlicNameRef: slicName}}
-			vmi.Spec.Volumes = []v1.Volume{
-				{
-					Name:         slicName,
-					VolumeSource: source,
-				},
+		DescribeTable("slic ACPI table should be set to", func(slicVol, msdmVol *v1.Volume, shouldFail bool) {
```

Comment:
Rename that table? I'd also be okay with extracting this into two individual tables.

Reply:
I moved it into two different It() tests. The former, SLIC, was kept while I've added the MSDM. I think its best indeed to not mix them. The binary blob is different (header) and it only complicate by trying to combine them.